### DISCUSSION TITLE
Error! Unsupported Builder version `undefined` from "@natanffs/vercel-python-wsgi" Error: Command "vercel build" exited with 1

### DISCUSSION THREAD

**Discussion by MyProgramsForTT** [vercel/vercel#8076] [Help] [answered]
Error! Unsupported Builder version `undefined` from "@natanffs/vercel-python-wsgi"
--
Error: Command "vercel build" exited with 1
我的文件是这样的
`{
  "version": 2,
  "builds": [
    {
      "src": "Django/wsgi.py",
      "use": "@natanffs/vercel-python-wsgi",
      "config": {
        "maxLambdaSize": "15mb",
        "runtime": "python3.9"
      }
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "Django/wsgi.py"
    }
  ]
}`

**Comment by maesterzak** [comment, marked as answer]
Check this post [Here](https://www.devmaesters.com/blog/15) 
I explained how to deploy a Django project on vercel.

**Comment by MyProgramsForTT** [comment]
Thanks for the pointers, sorry for not seeing your article earlier, I tried it and it worked, thanks.

发自我的iPhone

**Reply by maesterzak** [reply]
Your welcome 👍
